packages/template/package.json

```
@@ -11,16 +11,18 @@
     "watch": "bsb -make-world -clean-world -w"
   },
   "dependencies": {
-    "bs-platform": "^5.0.4",
-    "reason-react-native": "^0.60.0",
-    "expo": "^34.0.3",
-    "expo-linear-gradient": "^5.0.1",
-    "react": "16.8.3",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
-    "reason-expo": "^34.0.0",
+    "expo": "~36.0.0",
+    "react": "~16.9.0",
```

```suggestion
    "react": "16.9.0",
```

needs to be hardcoded